app/src/main/res/layout/main_fragment.xml

```diff
@@ -73,16 +73,16 @@
 
         <ImageButton
             android:id="@+id/btnTakePic"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="149dp"
```

Comment:
Avoid using fixed sizes in a constraint layout.  Constraint layouts are meant to be fluid, and look good on multiple screen densities.  If you want to change the proportion of widgets on a screen, consider using layout)weight, or set a preferred size.